### PR TITLE
Handle case when Clair API does not find open CVEs

### LINKED ISSUE #34: pierone fails with backtrace when the CVE status is COULDNT_FIGURE_OUT
```
Traceback (most recent call last):
  File "/usr/local/bin/pierone", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.4/dist-packages/pierone/cli.py", line 485, in main
    cli()
  File "/usr/local/lib/python3.4/dist-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.4/dist-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.4/dist-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.4/dist-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/click/decorators.py", line 27, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/pierone/cli.py", line 313, in cves
    installed_software = get_clair_features(config.get('clair_url'), artifact_tag.get('clair_id'), token)
  File "/usr/local/lib/python3.4/dist-packages/pierone/cli.py", line 235, in get_clair_features
    return r.json()['Layer']['Features']
KeyError: 'Features'
```

### PR DESCRIPTION
**Pull request by rafaelcaricio**
Fixes #34

### CODE REVIEW

File: pierone/cli.py [hunk 1]
```diff
@@ -232,7 +232,11 @@ def get_clair_features(url, layer_id, access_token):
         return []
     else:
         r.raise_for_status()
-    return r.json()['Layer']['Features']
+
+    payload = r.json()['Layer']
+    if 'Features' in payload:
+        return payload['Features']
```
Comment (hjacobs) [inline pierone/cli.py]: I actually prefer the one line version:

```
r.json()['Layer'].get('Features', [])

Comment (rafaelcaricio) [inline pierone/cli.py]: True! 😄

**Comment by hjacobs** [comment]
:+1:

**Comment by hjacobs** [comment]
Hmm, we need two approvals? Probably we should add a `.zappr.yaml` config file...

**Comment by rafaelcaricio** [comment]
👍

**Comment by dryewo** [comment]
👍

**Comment by rafaelcaricio** [comment]
Oh, I approved my own PR. :shipit:

**Pull request merged**